src/System.Diagnostics.FileVersionInfo/src/System/Diagnostics/FileVersionInfo.cs

```diff
@@ -260,7 +260,7 @@ public string SpecialBuild
         }
 
         /// <summary>
-        /// Returns a System.Windows.Forms.FileVersionInfo representing the version information associated with the specified file.
+        /// Returns a <see cref="T:System.Diagnostics.FileVersionInfo" /> representing the version information associated with the specified file.
```

Comment:
Yeah, the repo is currently very inconsistent in this regard, something to be cleaned up eventually. For now, since these references are inside the target type, I think it's cleanest to just use the shortest name possible. It'd be great if you could update it.